packages/react-app/src/App.jsx

```diff
@@ -82,6 +81,9 @@ const localProviderUrlFromEnv = process.env.REACT_APP_PROVIDER ? process.env.REA
 if (DEBUG) console.log("🏠 Connecting to provider:", localProviderUrlFromEnv);
 const localProvider = new ethers.providers.StaticJsonRpcProvider(localProviderUrlFromEnv);
 
+const deployerWallet = ethers.Wallet.fromMnemonic("test test test test test test test test test test test junk");
+const deployerSigner = deployerWallet.connect(localProvider);
```

Comment:
Here we initialize the signer for the deployer wallet so we can send the GTC tokens using the faucet transactor in line 463.